pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ojmicroline-thermostat"
-version = "3.2.0"
+version = "3.2.1"
```

```suggestion
version = "3.3.0"
```

A new feature will bump the version to 3.3.0 :)